### ISSUE TITLE
Add emmyft to AUTHORS

### ISSUE THREAD

**Issue by tobyhodges** [carpentries-incubator/jekyll-pages-novice#229]
After #228 we should add Emmy @emmyft to the author list for the lesson. @emmyft please feel free to submit another pull request to add your name to the AUTHORS file.

[edit: you are not obliged to include an email address in that file.]
